Server/BlazorRepl.Server.csproj

```diff
@@ -3,12 +3,20 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <DisableImplicitComponentsAnalyzers>true</DisableImplicitComponentsAnalyzers>
+    <BlazorWebAssemblyEnableLinking>false</BlazorWebAssemblyEnableLinking>
```

Comment:
Quick follow up to make sure I've got it right, we need 
BlazorWebAssemblyEnableLinking set true for all projects ?

Reply:
We don't need this option in any project except the Client project where we need it disabled for now. We will investigate reenabling this with a custom linker configuration but for not we need this disabled because it's trimming assemblies (or parts of some) that we need.
Sorry for not stating this in a clearer manner.

Reply:
**My bad** for not grasping it to begin with! 
Sorry for all the hassle , I hope it didn't added too much work

Reply:
Not at all! :) We are both very used to the code review process and we deeply believe it's always useful for producing working, human-understandable, elegant and optimal code and sharing knowledge across devs working on the project.
Thank you for being our first contributor!